lib/takeaway.rb

```diff
@@ -0,0 +1,82 @@
+require_relative 'twilio_sender'
+
+class Takeaway
+
+  attr_reader :menu, :twilio_sender
+  attr_accessor :selection
+
+  def initialize twilio_sender
+    @menu = {
+      pizza: 9.99,
+      chicken: 2.99,
+      kebab: 4.99,
+      beer: 0.99,
+      cider: 0.99,
+      wine: 7.99
+      }
+    @selection = {}
+    @twilio_sender = twilio_sender
+  end
+
+  def display_menu
+    @menu.each do |item, price|
+      "#{item.to_s.capitalize} : £#{price}"
+    end
+  end
+
+  def select_items item, quantity
+    if item_has_already_been_chosen? item
+      update_existing_quantity quantity, item
+    else
+      @selection[item] = quantity
+    end
+  end
+
+  def total
+    total_price = 0.0
+    selection.each { |item, quantity| get_price item
+      total_price += (@price * quantity) }
+    @price = sprintf('%.2f', total_price)
+    "£#{@price}"
+
+  end
+
+  def order to, amount
+    if amount == total
+      self.twilio_sender.send_text(to, order_message)
+    else
+      fail 'Cannot confirm this order'
+    end
+  end
+
+  private
+
+  attr_reader :menu
+
+  def get_price item
+    menu.select do |product, price|
```

Comment:
Unused block argument - `product`. You can omit all the arguments if you don't care about them.<br>Unused block argument - `price`. You can omit all the arguments if you don't care about them.